radio/src/gui/128x64/view_main.cpp

```diff
@@ -148,12 +148,13 @@ void doMainScreenGraphics()
 
 void displayTrims(uint8_t phase)
 {
+  bool squareMarker = (keysGetMaxTrims() <= 4);
   for (uint8_t i = 0; i < keysGetMaxTrims(); i++) {
 #if defined(SURFACE_RADIO)
-    static coord_t x[] = {TRIM_RH_X, TRIM_LH_X, TRIM_RV_X, TRIM_LV_X, TRIM_LV_X};
+    static uint8_t x[] = {TRIM_RH_X, TRIM_LH_X, TRIM_RV_X, TRIM_LV_X, TRIM_LV_X};
```

Comment:
Isn't the issue in coord_t type definition ? Need to try it by I feel it should be uint8_t for max(LCD_W, LCD_H) < 256

Reply:
coord_t can be outside screen bounds (negative and greater than width/height) so it can't be an unsigned value and 8 bits isn't enough.

I tried changing the type for coord_t to int16_t; but this increased code size by ~1K 😕

Reply:
Ok, not sure why it should allow off screen values tho, but thats probably opening another can of worm

Reply:
It looks like the trims are in the right place and all on the MT12... if I were to nitpick, there seems to be a (not related to this PR as it's present in main code also) issue with the trim 'knob' reaching the end at more like 80% travel... whereas on colorlcd the knob stops moving exactly at end of trim travel. This is without any extended trims, etc... new blank model.

Reply:
@3djc Let me know if you need/want time to look into the coord_t/uint8_t change further... otherwise I think this is ready to merge.

Reply:
> if I were to nitpick, there seems to be a (not related to this PR as it's present in main code also) issue with the trim 'knob' reaching the end at more like 80% travel... whereas on colorlcd the knob stops moving exactly at end of trim travel.

Fixed.

Reply:
LGTM on MT12 and Pocket. Thanks :)